infra/release/release_note.md

```diff
@@ -1,28 +1,46 @@
-# Release Note 0.2.0
+# Release Note 0.3.0
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- More commands are supported.
-  - [Open with Text Editor] command
-  - [Open Containing Folder] command
-  - [Collapse All] command
-- Improve [Create Config] command
+- Add hide-and-show button
+- Show more files
+  - memory trace files
+  - performance trace files
+  - circle log files
+  - extra backend model binary files
+- Open files with dedicated editor/viewers
+  - circle files - 'Circle Viewer'
+  - memory trace files - 'Mondrian Viewer'
+  - performance trace files - 'Json Tracer'
+  - circle log files - 'Text Editor'
 
 ### Toolchain View
 
-- Show Installable Toolchains on ToolchainView
-  - official release version
-  - nightly release version
-- Install Toolchain
-- Uninstall Toolchain
-- Show already-installed Toolchain
-- Support sudo command on Toolchain
+- TODO: Fill this part
+
+### Target Device View
+
+- Add Device list on local connection
+    - currently ubuntu 18.04 host PC and Tizen device added 
+- Show usable Executors under Device
 
 ### Cfg Editor
 
-- Editor is upgraded with whole new UI.
-- [Optimize] section is now supported.
-- [Quantize] section is now supported.
-- Bugfix
+- Help message is shown when mouse is on the question mark icon
+- Path of file can be set by clicking search icon
+- Trivial bugfix
+
+### Circle Graph Viewer
```

Comment:
For the name, 'Circle Graph Viewer' or 'Circle Viewer'?
I think the internal class name is Circle Graph Viewer but the registered viewer name is 'Circle Viewer'.
https://github.com/Samsung/ONE-vscode/blob/76d8e5221fb2cea8ac477e49191616f3fd6bca93/package.json#L83

Reply:
I made each section as following the source directory names. As you mentioned, `Circle Viewer` feels more appropriate term. @seanshpark, How do you think about this?

Reply:
`Circle Viewer` is good to me :)

Reply:
```suggestion
### Circle Viewer
```